vector/src/main/java/im/vector/adapters/VectorMessagesAdapter.java

```diff
@@ -1873,6 +1876,13 @@ private void manageSelectionMode(final View contentView, final Event event, fina
             tsTextView.setVisibility(View.VISIBLE);
         }
 
+        StickerMessage stickerMessage = JsonUtils.toStickerMessage(event.getContent());
```

Comment:
Should be done after having checked Event.EVENT_TYPE_STICKER.

Reply:
you have msgType

Reply:
It's not a msgType from server but a new one in local to hack json parsing. It was created in order to obtain a StickerMessage even if it's an event type. 
It allows us to have a StickerMessage that inherits from ImageMessage. However, you can not use the msgType to do the checks.